src/shims/ApiCompat.proj

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="dir.props" />
+
+  <PropertyGroup>
+    <ApiCompatResponseFile>$(IntermediateOutputPath)/apicompat.rsp</ApiCompatResponseFile>
+    <ApiCompatBaselineFile>$(MSBuildThisFileDirectory)ApiCompatBaseline.$(TargetGroup).netfx461.txt</ApiCompatBaselineFile>
+    <ApiCompatNSBaselineFile>$(MSBuildThisFileDirectory)ApiCompatBaseline.$(TargetGroup).netstandard20.txt</ApiCompatNSBaselineFile>
+    <ApiCompatImplementationDirs>$(RefPath),$(GenFacadesOutputPath)</ApiCompatImplementationDirs>
+    <!-- If we are targeting uap or uapaot run ApiCompat against the implementation assemblies instead since they don't match the reference assemblies -->
+    <ApiCompatImplementationDirs Condition="'$(TargetGroup)'=='uap' or '$(TargetGroup)'=='uapaot'">$(RuntimePath),$(GenFacadesOutputPath)</ApiCompatImplementationDirs>
+  </PropertyGroup>
+
+  <!-- Run ApiCompat -->
+  <Target Name="RunApiCompat"
+          Inputs="$(ApiCompatResponseFile);@(GenFacadesContracts)"
+          Outputs="$(ApiCompatBaselineFile);$(ApiCompatNSBaselineFile)"
+  >
+    <PropertyGroup>
+      <ApiCompatArgs>$(ApiCompatArgs) -implDirs:"$(ApiCompatImplementationDirs)"</ApiCompatArgs>
+      <ApiCompatExitCode>0</ApiCompatExitCode>
+
+      <ApiCompatCmd>$(ToolHostCmd) "$(ToolsDir)ApiCompat.exe"</ApiCompatCmd>
+    </PropertyGroup>
+
+    <MakeDir Directories="$(IntermediateOutputPath)" />
+    <WriteLinesToFile File="$(ApiCompatResponseFile)" Lines="$(ApiCompatArgs)" Overwrite="true" />
+
+    <Exec Command="$(ApiCompatCmd) &quot;@(NetFxReference->'$(NetFxRefPath)%(Identity).dll')&quot; @&quot;$(ApiCompatResponseFile)&quot; &gt; $(ApiCompatBaselineFile)"
```

Comment:
Now that we are running this potentially on multiple frameworks we should consider changing it to a batching model like we have in standard (https://github.com/dotnet/standard/blob/master/apicompat/apicompat.proj#L193).